src/RtcDS3231.cpp

```diff
@@ -123,7 +123,7 @@ RtcDateTime RtcDS3231::GetDateTime()
     uint8_t minute = BcdToUint8(Wire.Read());
     uint8_t hour = BcdToBin24Hour(Wire.Read());
 
-    Wire.Read();  // throwing away day of week as we calculate it
+    uint8_t dayOfWeek = BcdToUint8(Wire.Read());
```

Comment:
The reason to ignore the stored day of week from the RTC, is that it can be set incorrectly by others as the field is not checked inside the device for accuracy; its just a field that is incremented on the 0th hour.  
The calculation for the day of the week isn't very complex and removes this possible error.
What is the reason you want to include getting the day of the week?